Specs/Scene/AxisSpec.js

```diff
@@ -0,0 +1,76 @@
+/*global defineSuite*/
+defineSuite([
+    'Scene/Axis',
+    'Core/Cartesian4',
+    'Core/Math',
+    'Core/Matrix3',
+    'Core/Matrix4'
+], function(
+    Axis,
+    Cartesian4,
+    CesiumMath,
+    Matrix3,
+    Matrix4) {
+    'use strict';
```

Comment:
Fix the indentation slightly: https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Specs/Core/sampleTerrainSpec.js#L2-L10